packages/validate-commits/lib/reporter.js

```diff
@@ -1,21 +1,22 @@
-'use strict';
-
 const chalk = require('chalk');
 
 class Reporter {
-    printReport(report) {
-        report.commits.map(commit => {
-            if (commit.valid) {
-                return console.log(chalk.green(commit.subject));
-            }
+  printReport(report) {
+    report.commits.map((commit) => {
+      if (commit.valid) {
+        // eslint-disable-next-line no-console
+        return console.log(chalk.green(commit.subject));
+      }
 
-            return console.log(chalk.red.bold(commit.subject));
-        });
+      // eslint-disable-next-line no-console
+      return console.log(chalk.red.bold(commit.subject));
+    });
 
-        if (report.message) {
-            console.log(report.message);
-        }
+    if (report.message) {
+      // eslint-disable-next-line no-console
```

Comment:
🍹 Since this repo implements a reporter, maybe its worth it to disable this rule for the whole repo / file.

Reply:
I'm going to get rid of the reporter in a later commit, so there's really only going to be one log, here:
https://github.com/bigcommerce/validate-commits/blob/f2d1c2604e703212ae7b4d48cdaaef2241053158/packages/validate-commits/bin/validate-commits.js#L26